modern/src/common/formatter.js

```diff
@@ -9,7 +9,7 @@ export const formatPosition = (value, key) => {
     case 'fixTime':
     case 'deviceTime':
     case 'serverTime':
-      return moment(value).format('LLL');
+      return moment(value).format('YYYY-MM-DD HH:mm');
```

Comment:
This is not a universal solution. We need a better one.

Reply:
Changes reverted. Tried a new solution.